static/app/types/core.tsx

```diff
@@ -125,10 +129,14 @@ export interface DataCategoryInfo {
   name: DataCategoryExact;
   plural: DataCategory;
   productName: string;
+  statsInfo: {
+    showExternalStats: boolean;
+    showInternalStats: boolean;
+    yAxisMinInterval: number;
+  };
   titleName: string;
   uid: number;
   docsUrl?: string;
-  snakeCasePlural?: string;
```

Comment:
i removed this because you can just extrapolate it from `.plural` with the lodash function `snakeCase`